React/Views/RCTWebView.m

```diff
@@ -92,6 +92,16 @@ - (void)setHTML:(NSString *)HTML
   [_webView loadHTMLString:HTML baseURL:nil];
 }
 
+- (void)setDecelerationNormalEnabled:(BOOL *)decelerationNormalEnabled
+{
+  // The default decelerationRate is UIScrollViewDecelerationFast. Allow user
+  // to override and set to normal in order to give the WebView the same
+  // "momentum" scrolling as other iOS views.
+  if (decelerationNormalEnabled) {
+    _webView.scrollView.decelerationRate = UIScrollViewDecelerationRateNormal;
+  }
+}
```

Comment:
you need to unset it if decelerationNormalEnabled is false. The code as is will never remove it once it's set